### DISCUSSION TITLE
access /sdcard of shelter work profile in termux on rooted phone

### DISCUSSION THREAD

**Discussion by aa755** [termux/termux-app#2426] [General]
I put my trusted applications in a work profile. they can communicate securely using the /sdcard of wrork profile without the fear of untrusted apps reading the contents. these trusted apps are all open source. I trust termux as well but I cannot install it in the work profile because of [this issue](https://github.com/termux/termux-app/issues/2168).
So my termux runs in the regular (non-work) profile unfortunately.
Fortunately, my phone is rooted. Can I somehow get termux to access the /sdcard of work profile?

**Comment by Seanld** [comment]
Not sure if you found it by now, but it looks like the filesystems are stored in /data/media for each profile on the system. That means the main user, and the work profile, and probably whatever other users you have added to your system if your specific flavor of Android supports that. They're all in separate directories. You'll have to look through each one to know which one's which, but that should be pretty simple.

**Reply by sylirre** [reply]
Yes, work profile is in /data/media. However files there are sensitive to set permissions and SELinux, as its easy to get files unreadable by applications accidentally.

**Comment by ChiefMikeK** [comment]
you could possibly modify this f-droid app  [app/src/main/java/com/gitlab/giwiniswut/rwremount/MountReceiver.java · main · Sebastian / ROOT - Android SD-Card RW Remounter · GitLab](https://gitlab.com/giwiniswut/android-media-rw-remounter/-/blob/main/app/src/main/java/com/gitlab/giwiniswut/rwremount/MountReceiver.java ) please read the well documented source code
